### DISCUSSION TITLE
CANBUS data stops capturing / freezes in Main capture window - Original post under issues #662

### DISCUSSION THREAD

**Discussion by itaintpriddy** [collin80/SavvyCAN#666] [Q&A]
Hey all

just revisiting this specific issue, my EVTV ESP32 Candue board will connect fine to SavvyCAN, but will freeze after a certain number of frames.  I removed a termination resistor on the board as I was getting a reading of 40ohms and not 60ohms across the whole bus and thought this may have been the issue. A quick look at the connection console...

Write to serial -> f1 9 
Write to serial -> f1 9 
Write to serial -> f1 9 
Got data from serial. Len = 4
Got data from serial. Len = 4
Got data from serial. Len = 4
f1 09 de ad 
f1 09 de ad 
f1 09 de ad 
Got data from serial. Len = 111
Got data from serial. Len = 111
Got data from serial. Len = 111
f1 00 2f 43 ac 02 16 02 00 00 02 40 24 00 f1 00 34 43 ac 02 15 02 00 00 06 ff f0 ff 00 ff ff 00 f1 00 39 43 ac 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 48 43 ac 02 80 02 00 00 08 01 ff c0 00 00 00 00 00 00 f1 00 4d 43 ac 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 53 43 ac 02 55 03 00 00 07 00 00 ff ff 00 00 40 00 
f1 00 2f 43 ac 02 16 02 00 00 02 40 24 00 f1 00 34 43 ac 02 15 02 00 00 06 ff f0 ff 00 ff ff 00 f1 00 39 43 ac 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 48 43 ac 02 80 02 00 00 08 01 ff c0 00 00 00 00 00 00 f1 00 4d 43 ac 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 53 43 ac 02 55 03 00 00 07 00 00 ff ff 00 00 40 00 
f1 00 2f 43 ac 02 16 02 00 00 02 40 24 00 f1 00 34 43 ac 02 15 02 00 00 06 ff f0 ff 00 ff ff 00 f1 00 39 43 ac 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 48 43 ac 02 80 02 00 00 08 01 ff c0 00 00 00 00 00 00 f1 00 4d 43 ac 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 53 43 ac 02 55 03 00 00 07 00 00 ff ff 00 00 40 00 
Got data from serial. Len = 112
Got data from serial. Len = 112
Got data from serial. Len = 112
f1 00 cf c9 ad 02 16 02 00 00 02 40 24 00 f1 00 d4 c9 ad 02 15 02 00 00 06 ff f0 ff 00 ff ff 00 f1 00 d9 c9 ad 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 e7 c9 ad 02 80 02 00 00 08 01 ff c0 00 00 00 00 00 00 f1 00 ec c9 ad 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 f2 c9 ad 02 5d 03 00 00 08 00 03 00 00 00 00 00 00 00 
f1 00 cf c9 ad 02 16 02 00 00 02 40 24 00 f1 00 d4 c9 ad 02 15 02 00 00 06 ff f0 ff 00 ff ff 00 f1 00 d9 c9 ad 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 e7 c9 ad 02 80 02 00 00 08 01 ff c0 00 00 00 00 00 00 f1 00 ec c9 ad 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 f2 c9 ad 02 5d 03 00 00 08 00 03 00 00 00 00 00 00 00 
f1 00 cf c9 ad 02 16 02 00 00 02 40 24 00 f1 00 d4 c9 ad 02 15 02 00 00 06 ff f0 ff 00 ff ff 00 f1 00 d9 c9 ad 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 e7 c9 ad 02 80 02 00 00 08 01 ff c0 00 00 00 00 00 00 f1 00 ec c9 ad 02 de 02 00 00 08 00 00 80 05 d0 00 ff ff 00 f1 00 f2 c9 ad 02 5d 03 00 00 08 00 03 00 00 00 00 00 00 00 

If the vehicle ignition is switched to "ACC" mode, it will only capture between 800 - 3000 frames, frames per second is approx 50-70. When I then tried resetting everything, let it capture approx 300 frames and then turn the ignition from "ACC" to "ON" position, the frames will stop capturing almost immediately. 

![image](https://github.com/collin80/SavvyCAN/assets/35412401/845f28e3-0e10-410d-b0d5-c312c5a8e636)

I have tried this on another vehicle and same thing occurs,
Ran this Dual boot on Windows 11 and on Ubuntu, but getting the same result. I also backtracked to V199 and used the continuous V213 to no avail. I also used a shorter cable (stub) to connect the OBD2, hardwired directly to the vehicle, changed 3 USB cables and still no go.

A little stuck here, if anyone may have any ideas or suggestions of what else I could be doing, please let me know.

Appreciate any assistance

**Comment by collin80** [comment]
You seem to have covered quite a few of the ideas I would otherwise suggest. I'm not really sure what is going on. You might try connecting to the device with a serial terminal program like CoolTerm or PuTTY and see if the serial output there can stay active and not freeze up.

**Comment by xnlfgh** [comment]
I am having a very similar problem so i hope someone has an answer that helps both of us...
